selfdrive/car/nissan/values.py

```diff
@@ -20,29 +19,52 @@ def __init__(self, CP):
     pass
 
 
-class CAR(StrEnum):
-  XTRAIL = "NISSAN X-TRAIL 2017"
-  LEAF = "NISSAN LEAF 2018"
-  # Leaf with ADAS ECU found behind instrument cluster instead of glovebox
-  # Currently the only known difference between them is the inverted seatbelt signal.
-  LEAF_IC = "NISSAN LEAF 2018 Instrument Cluster"
-  ROGUE = "NISSAN ROGUE 2019"
-  ALTIMA = "NISSAN ALTIMA 2020"
-
-
 @dataclass
 class NissanCarInfo(CarInfo):
   package: str = "ProPILOT Assist"
   car_parts: CarParts = field(default_factory=CarParts.common([CarHarness.nissan_a]))
 
 
-CAR_INFO: dict[str, NissanCarInfo | list[NissanCarInfo] | None] = {
-  CAR.XTRAIL: NissanCarInfo("Nissan X-Trail 2017"),
-  CAR.LEAF: NissanCarInfo("Nissan Leaf 2018-23", video_link="https://youtu.be/vaMbtAh_0cY"),
-  CAR.LEAF_IC: None,  # same platforms
-  CAR.ROGUE: NissanCarInfo("Nissan Rogue 2018-20"),
-  CAR.ALTIMA: NissanCarInfo("Nissan Altima 2019-20", car_parts=CarParts.common([CarHarness.nissan_b])),
-}
+@dataclass
+class NissanCarSpecs(CarSpecs):
+  centerToFrontRatio: float = field(default=0.44)
+  steerRatio: float = field(default=17.)
```

Comment:
```suggestion
  centerToFrontRatio: float = 0.44
  steerRatio: float = 17.
```

I believe you don't need to use field when dealing with immutable default types (same as default function arg values)

Reply:
done